lib/swagger.js

```diff
@@ -68,6 +68,11 @@
     };
   }
 
+  function normalizeTextValue(value) {
+    return Object.prototype.toString.call(value) === '[object Array]' ?
```

Comment:
This is a polyfill implementation of `Array.isArray` from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray#Polyfill